frontend/scss/components/organisms/header.scss

```
@@ -81,53 +96,256 @@
           @include txt-lrg;
         }
       }
-
-      &-sub-title {
-        color: color('black');
-        white-space: nowrap;
-        text-overflow: ellipsis;
-        overflow: hidden;
-      }
   }
 
-  &-first-level {
+  &-main {
     display: none;
     flex: 1 1 auto;
-    padding-left: 30px;
-    padding-right: 30px;
-    padding-bottom: 10px;
+    padding: 0 30px;
 
     @media (min-width: 1024px) {
       display: flex;
     }
 
+    &-item {
+      display: flex;
+      align-items: center;
+      margin: 0 18px -5px;
+      padding-bottom: 14px;
+    }
+
+    &-item:hover {
+
+      & > .ap-o-header-flyout {
+          opacity: 1;
+          pointer-events: all;
+          transition: opacity 0.2s cubic-bezier(0.25, 0.1, 0.25, 1);
+
+          &-secondary {
+            z-index: -1;
+          }
+      }
+    }
+
     &-link {
-      margin: 0 20px;
+      border-radius: 10px;
 
-      &:last-child {
-        margin-left: auto;
+      &-icon {
+        font-size: 8px;
+        margin-left: 5px;
+        transform: rotate(0deg);
+        transition: transform 0.2s cubic-bezier(.25,.1,.25,1);
+
+        &.right {
+          transform: rotate(-90deg);
+          margin-left: auto;
+          font-size: 10px;
+        }
       }
     }
   }
 
-  &-second-level {
-    display: none;
-    flex: 1 1 100%;
-    padding-top: 8px;
-    /* Pull out to have a 100% width border */
-    margin-left: -30px;
-    padding-left: 146px;
-    padding-bottom: 8px;
-    border-top: 1px solid color('wild-sand');
+  &-flyout {
+    @include flayout-list-shadow;
+    pointer-events: none;
+    opacity: 0;
+    list-style: none;
+    background: #fff;
+    position: absolute;
+    top: 100%;
+    padding: 0 0 10px;
+    margin-left: 0;
+    border-bottom-right-radius: 4px;
+    border-bottom-left-radius: 4px;
+    transition: opacity 0.2s cubic-bezier(0.25, 0.1, 0.25, 1);
 
-    @media (min-width: 1024px) {
-      display: block;
+    &-fake-shadow {
+      width: 100%;
+      background: linear-gradient(to bottom, rgba(0, 0, 0, 0.07) 0%,#ffffff 100%);
```

Please use the color method and map.

```suggestion
      background: linear-gradient(to bottom, fade_out(color('black'), 0.93) 0%, color('white') 100%);
```